docs/plugins/available-plugins.md

```diff
@@ -73,125 +102,173 @@ tests | <b>80</b> | The more tests a plugin has, the better
 
 -----
 
-<div class="quality-index" style="color: #72CC1D">526</div>
+<div class="quality-index" style="color: #72CC1D">469</div>
 
-<h3><a href="https://github.com/joshdholtz/fastlane-plugin-s3" target="_blank"><code>aws_s3</code></a></h3>
+<h3><a href="https://github.com/KrauseFx/trainer" target="_blank"><code>trainer</code></a></h3>
 
-<p class="via-text">via Josh Holtz</p>
+<p class="via-text">via KrauseFx</p>
+
+> Convert xcodebuild plist files to JUnit reports
+
+
+<details>
+<summary class="plugin-actions">Provided actions</summary>
+
+ Name | Category | Description
+------|----------|------------
+  **trainer** | - | Convert the Xcode plist log to a JUnit report. This will raise an exception if the tests failed
+
+</details>
 
-> Upload IPA and APK to S3
 
 <details>
 <summary class="score-details">Score details</summary>
 
  Metric | Points | Description
 --------|--------|-------------
-contributors | <b>72</b> | The more contributors a project has, the more likely it is it stays alive
-subscribers | <b>9</b> | More subscribers = more popular project
-stars | <b>27</b> | More stars = more popular project
-forks | <b>90</b> | More forks = more people seem to use/modify this project
+contributors | <b>0</b> | The more contributors a project has, the more likely it is it stays alive
```

Comment:
this looks strange, for multiple plugins

Reply:
Hmm, I didn't change anything in the code that computes these scores: https://github.com/fastlane/fastlane/pull/11117/files

Don't know where this could come from

Reply:
Environment thing maybe? API not responding?

Reply:
(Or again only a mismatch in the diff?)

Reply:
I thought that too at first, but all the other values from the GitHub API seem to be filled in correctly